inc/Engine/CDN/CDN.php

```diff
@@ -207,7 +205,7 @@ public function get_cdn_urls( $zones = 'all' ) {
 	 * @return string
 	 */
 	private function get_base_url() {
-		return '//' . wp_parse_url( get_option( 'home' ), PHP_URL_HOST );
+		return '//' . wp_parse_url( site_url(), PHP_URL_HOST );
```

Comment:
Let's talk about `site_url()` vs. `home_url()` to replace `get_option( 'home' )`:

- WordPress' `home_url()` function grabs `get_option( 'home' ):
```php
	if ( empty( $blog_id ) || ! is_multisite() ) {
		$url = get_option( 'home' );
	} else {
		switch_to_blog( $blog_id );
		$url = get_option( 'home' );
		restore_current_blog();
	}
```
- Polylang, WPML, and qTranslate all use the filter `'home_url`' and none use `'site_url'`
- These same i18n plugins also use `home_url()`
- It seems all of them use `home_url()` and `'home_url'` filter for getting or setting the right language

@Tabrisrp It seems `home_url()` is more aligned with the original functionality than `site_url()`. 

Switching this code from `get_option( 'home' )` to `home_url()` is a good strategy to fire the filter which then triggers the language plugins and anything else that filters the home URL.

Reply:
> Polylang, WPML, and qTranslate all use the filter 'home_url' and none use 'site_url'

That's not entirely true. Polylang definitely filters `site_url` (2 occurrences in the current code), WPML too since we tested it on a customer website above.

`site_url` is used in WP as the base URL for anything in `wp_includes|wp-content|wp-content/uploads`, so it made sense to me to use it. But in the end the `host` part of either `home_url()` or `site_url()` should be the same, so I don't mind switching to `home_url()` there. Going to make the change in a bit.

Reply:
https://github.com/wp-media/wp-rocket/pull/2589/commits/b587bf701af10ed681825e8218d9f6f7a03ade54